pkg/kubelet/apis/kubelet.config.k8s.io/types.go

```diff
@@ -63,8 +62,7 @@ type KubeletConfiguration struct {
 	HTTPCheckFrequency metav1.Duration
 	// manifestURL is the URL for accessing the container manifest
 	ManifestURL string
-	// manifestURLHeader is the HTTP header to use when accessing the manifest
-	// URL, with the key separated from the value with a ':', as in 'key:value'
+	// manifestURLHeader is the HTTP header to use when accessing the manifestURL
```

Comment:
is a map of slices with HTTP headers to use when accessing the manifestURL

Reply:
done